src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -97,3 +97,11 @@ def transform_web_list_output(webs):
 
 cli_command(__name__, 'appservice web deployment user show', 'azure.mgmt.web.web_site_management_client#WebSiteManagementClient.get_publishing_user', cf_web_client, exception_handler=empty_on_404)
 cli_command(__name__, 'appservice list-locations', 'azure.mgmt.web.web_site_management_client#WebSiteManagementClient.list_geo_regions', cf_web_client, transform=transform_list_location_output)
+
+cli_command(__name__, 'functionapp create', 'azure.cli.command_modules.appservice.custom#create_function')
+cli_command(__name__, 'functionapp list', 'azure.cli.command_modules.appservice.custom#list_functionapp', table_transformer=transform_web_list_output)
+cli_command(__name__, 'functionapp show', 'azure.cli.command_modules.appservice.custom#show_functionapp', exception_handler=empty_on_404, table_transformer=transform_web_output)
+cli_command(__name__, 'functionapp delete', 'azure.cli.command_modules.appservice.custom#delete_functionapp')
+cli_command(__name__, 'functionapp stop', 'azure.cli.command_modules.appservice.custom#stop_functionapp')
+cli_command(__name__, 'functionapp start', 'azure.cli.command_modules.appservice.custom#start_functionapp')
+cli_command(__name__, 'functionapp restart', 'azure.cli.command_modules.appservice.custom#restart_functionapp')
```

Comment:
I would argue we don't. People are going to be confused when `functionapp list` returns more than function apps, etc. etc.  The different is so slight I don't see why these aren't bundled together.